app/src/main/java/com/example/pygmyhippo/common/Entrant.java

```diff
@@ -21,17 +21,21 @@
  * @author James Fu, Griffin, Katharine
  */
 public class Entrant {
-
     private String accountID;
     private EntrantStatus entrantStatus;
+
+    @Nullable
+    private EntrantStatus notifiedStatus;
+
     private Double longitude;
     private Double latitude;
 
     public enum EntrantStatus {
         waitlisted("waitlisted"),
         cancelled("cancelled"),
         invited("invited"),
-        accepted("accepted");
+        accepted("accepted"),
+        rejected("rejected");
```

Comment:
in your mind, does the rejected entrant have a second chance at being selected from the waitlist (as they lost the lottery) or are they just off the waitlist now?

Reply:
I think they should be off the waitlist (unless theres a user story that says otherwise). I think it should be treated the same as a user leaving the waitlist

Reply:
Its hard for the notifications implementation to know to send a "lost lottery" notification if the entrant is removed. So rejected was created for that. If we are going to remove entrants that lost the lottery then notifications would have to be changed greatly along with some db schemas

Reply:
let's talk tmmr, i just want to know how to handle it in view my single event on the user side :P